scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestController_ScalingEngine_IntegrationTest.java

```diff
@@ -0,0 +1,186 @@
+package org.cloudfoundry.autoscaler.scheduler.rest;
```

Comment:
Integration tests should live elsewhere. We need to run units and know they succeed before we attempt integration tests.